polaris-migrator/src/migrations/styles-insert-stylelint-disable/styles-insert-stylelint-disable.ts

```diff
@@ -0,0 +1,126 @@
+import type {FileInfo} from 'jscodeshift';
+import postcss, {Node, Comment, Plugin} from 'postcss';
+import stylelint, {StylelintPostcssResult} from 'stylelint';
+
+const crossPlatformNewlineRegExp = /\r\n?|\n/;
+
+const plugin = (): Plugin => {
+  const polarisContextMsg = '-- generated by polaris-migrator DO NOT COPY';
+
+  return {
+    postcssPlugin: 'insert-stylelint-disable',
+    Once(root, {result}) {
+      // Fill descriptionless disables with generated descriptions
+      // if reportDiscriptionlessDisables is true
+      const stylelintResult = (result as any)
+        ?.stylelint as StylelintPostcssResult;
+
+      if (stylelintResult?.config?.reportDescriptionlessDisables) {
+        root.walkComments((comment) => {
+          if (!comment.text.includes('--')) {
+            comment.text = [comment.text, polarisContextMsg].join(' ');
+          }
+        });
+      }
```

Comment:
I couldn't write a test for this case yet because it will only be enabled once [stylelint-polaris v5](https://github.com/Shopify/polaris/pull/7691/files) ships. I tested manually though on a v5 branch